terrain_planner/src/terrain_ompl_rrt.cpp

```diff
@@ -37,7 +37,7 @@
 // Constructor
 TerrainOmplRrt::TerrainOmplRrt() {
   problem_setup_ =
-      std::make_shared<ompl::OmplSetup>(ompl::base::StateSpacePtr(new fw_planning::spaces::DubinsAirplaneStateSpace()));
+      std::make_shared<ompl::OmplSetup>(ompl::base::StateSpacePtr(new ompl::base::OwenStateSpace(66.7, 0.3 * M_PI)));
```

Comment:
Do you intend to hard code these?

Reply:
It was taking the default value anyway. I can modify it such that it takes the default value on declaration.

Reply:
Done!